### PR TITLE
in progress fix

### PR DESCRIPTION
**Pull request by naomigreen**
**What**  
{summarise your PR and what it is for, add a screenshot for UI changes}
In progress fix and search label fix 

**Why**  
{why are we making this change?}

**Anything else?**

- Have you added any new third-party libraries?
- Are any new environment variables or configuration values needed?
- Anything else in this PR that isn't covered by the what/why?

### CODE REVIEW

File: components/Search/Search.jsx [hunk 1]
```diff
@@ -118,7 +118,8 @@ const Search = ({ query, type }) => {
             <>
               <div className="lbh-table-header">
                 <h2 className="govuk-fieldset__legend--m govuk-custom-text-color">
-                  {type.toUpperCase()} SEARCH RESULT
+                  {type === 'cases' ? 'RECORDS' : type.toUpperCase()} SEARCH
```
Comment (sirLisko) [inline components/Search/Search.jsx:121]: Can we rename the `type` instead?

Comment (naomigreen) [reply components/Search/Search.jsx:121]: We're using type for the people label so we need to keep it 
Screenshot 2020-12-16 at 10 28 03

Comment (sirLisko) [reply components/Search/Search.jsx:121]: yes, what I meant is that we are defining `type="cases"` in https://github.com/LBHackney-IT/lbh-social-care/blob/create_new_user/pages/cases/index.jsx#L9. We should make that `records`.

File: pages/cases/index.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ const SearchCasesPage = ({ query }) => {
   return (
     <div>
       <NextSeo title="Find unlinked case notes" noindex />
-      <Search query={query} type="cases" />
+      <Search query={query} type="Records" />
```
Comment (sirLisko) [inline pages/cases/index.jsx:9]: Can we have `records` lowercase so we are consistent.
You also need to change the checks `type === "cases"` in the `Search` component.

File: components/Layout/Header/Header.jsx [hunk 3]
```diff
@@ -4,8 +4,9 @@ import Link from 'next/link';
 import { useRouter } from 'next/router';
 
 import { useContext } from 'react';
-import UserContext from 'components/UserContext/UserContext';
 import { getDataIncludes } from 'utils/saveData';
+
+import UserContext from 'components/UserContext/UserContext';
```
Comment (sirLisko) [inline components/Layout/Header/Header.jsx:9]: Can we roll back these changes that are confusing? Same for the `_app` component

**Review by sirLisko — CHANGES_REQUESTED** [review]

**Review by sirLisko — APPROVED** [review]
👍

**Pull request merged**